deploy/addons/gcp-auth/gcp-auth-webhook.yaml.tmpl

```diff
@@ -61,6 +61,7 @@ metadata:
   name: gcp-auth-certs-create
   namespace: gcp-auth
 spec:
+  ttlSecondsAfterFinished: 30
```

Comment:
long term it would be better to have the replace the job instead of hoping it gets deleted on time before refresh but I understand this is a less invasive fix

Reply:
this is probably a good idea regardless of a longer term fix, but yeah, there should be a more stable solution to avoid colliding jobs